miner/README.org

```diff
@@ -92,7 +92,7 @@ Doing so, we'll see the following configuration of your "mining identity":
     predicate: keys-all
 #+end_src
 
-With values specific to you replaced above (and the rest of Chainweb configured
+With values specific to you replaced above (and the rest of the Kadena public blockchain configured
```

Comment:
In this case, the software is called `chainweb-node`, and that's what we mean here. The "Kadena public blockchain" is the entire network that exists "in the cloud" as a result of everyone running `chainweb-node`s.

Reply:
Many of the changes from "Chainweb" to "the Kadena public blockchain" here make sense to me as a branding move.  But this one in particular seems weird.

Reply:
Then to get rid of the ambiguity completely we can say `chainweb-node` directly.